build_runner/test/daemon/daemon_test.dart

```diff
@@ -191,9 +191,9 @@ main() {
       await d.dir('a', [
         d.dir('web', [
           d.file('main.dart', '''
-                main() {
-                  print('hello world');
-                }'''),
+main() {
+  print('goodbye world');
```

Comment:
Dark...

Reply:
lol I could go for the exclamation point instead